content/study-configuration/destination.md

```diff
@@ -4,28 +4,44 @@ date: 2023-04-05T08:41:03+02:00
 draft: false
 ---
 
-Every study needs a destination, where do the recruitment ads send the users? Destinations need to be connected to Virtual Lab so that, not only does it know where to send people who click on the recruitment ads, but also so that it knows how to collect the information about those who become study participants and knows how to optimize ads.
+Every study needs a destination, where do the recruitment ads send the users to? 
 
-Towards that end, Virtual Lab supports a set a "destinations" and is written in such a way that it is easy to add a new destination. When configuring your study, you can create one or more destinations and each destination needs a unique name (key) that you can refer to it by.
+Destinations need to be connected to Virtual Lab so that, not only does it know where to send people who click on the recruitment ads, but also so that it knows how to collect the information about those who become study participants and knows how to optimize ads.
 
-The following are currently supported along with their method of configuation:
+Towards that end, Virtual Lab supports a set of "destinations" and is written in such a way that it is easy to add a new destination. When configuring your study, you can create one or more destinations and each destination needs a unique name (key) that you can refer to it by.
 
-Fly Messenger Survey Destination. This creates "Messenger ads" in Facebook for when you want the user to be directed to a Fly Messenger Survey. It requires the following parameters:
+There are currently three types of destinations available:
 
-- `initial_shortcode`: str The shortcode of the initial form users should be sent to.
-- `survey_name`: str The survey that contains all the forms with data of interest.
+1. [Fly Messenger Survey](#fly-messenger-survey)
+2. [Typeform](#typeform)
+3. [Curious Learning Literacy App](#curious-learning-literacy-app)
 
-Typeform. Each Typeform destination needs the following parameters:
+## Fly Messenger Survey 
 
-- `form_id`: str The Form ID of the Typeform you wish to send users to.
+This creates "Messenger ads" in Facebook for when you want the user to be directed to a Fly Messenger Survey. 
 
-Curious Learning Literacy Apps. Requires the following parameters:
+We require the following to be set:
 
-- `from`: int Starting timestamp for retrieving app events from Literacy Data API.
-- `app_id`: str The app ID for the Google Play store.
-- `facebook_app_id`: str he app ID for Facebook.
+- `Initial Shortcode`: The shortcode of the initial form users should be sent to.
+- `Survey Name`: The survey that contains all the forms with data of interest.
+
+## Typeform
```

Comment:
@nandanrao is this still valid or do we now consider it a Web APP Destination with a field of `URL Template`